### PR TITLE
ARTEMIS-4498 enable management for internal addresses & queues

### PR DESCRIPTION
**Pull request by jbertram**

### CODE REVIEW

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java [hunk 1]
```diff
@@ -1881,7 +1904,8 @@ public int getEncodeSize() {
          BufferHelper.sizeOfNullableInteger(idCacheSize) +
          BufferHelper.sizeOfNullableSimpleString(pageFullMessagePolicy != null ? pageFullMessagePolicy.toString() : null) +
          BufferHelper.sizeOfNullableInteger(prefetchPageBytes) +
-         BufferHelper.sizeOfNullableInteger(prefetchPageMessages);
```
Comment (clebertsuconic) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java:1884]: why? I added a new record for AddressSettings, right?

Comment (clebertsuconic) [reply artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java:1884]: I wouldn't bother about this.. it's not going to be used.. the new record would be used.


Although shouldn't we always enable management for internal addresses?

**Comment by clebertsuconic** [comment]
Can't we just always allow management of internal queues? that was my point on the other PR.

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java (hunk 1 above)
Comment (jbertram) [reply artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java:1884]: Indeed. I'll remove those changes.

**Comment by jbertram** [comment]
After thinking about this more I think we should go with #4670.

**Pull request closed without merging**